src/App.js

```diff
@@ -1,16 +1,19 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- Hey, please, kindly remove this eslint disabling, it does not affect your code and it is not a good practice disabling the linter, this check helps us write clean and maintainable code.